ShowPot/ShowPot/Presentation/Scene/Login/CustomView/SocialLoginButton.swift

```diff
@@ -26,53 +26,99 @@ final class SocialLoginButton: UIButton {
     }
     
     private func setupStyles() {
-        var configuration = setButtonConfiguration(with: type)
-        var attributedTitle = setButtonAttributedString(with: type)
+        var configuration = createButtonConfiguration(with: type)
+        let attributedTitle = configureButtonAttributedString(with: type)
         configuration.attributedTitle = attributedTitle
```

Comment:
`createButtonConfiguration(with: type)`에서 `NSAttributedString`을 매개변수로 받아 처리하면 코드라인이 줄어들 것 같아요
```suggestion
        let attributedTitle = configureButtonAttributedString(with: type)
        let configuration = createButtonConfiguration(with: type, text: attributedTitle)
```

Reply:
넵 바로 수정해볼게요 !